_posts/sean-has-a-crystal-ball.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+header_image: /img/posts/sean-has-a-crystal-ball/header.jpg
+title: "Sean has a crystal ball"
```

Comment:
Consider a more descriptive title. Sean isn't mentioned until near the end and much of the article describes feedback mechanisms which will help people decide if they want to read it or not.